  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_arm_base
    pid: {p: 2.0, i: 0.1, d: 0.02}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_arm1
    pid: {p: 30.0, i: 0.1, d: 0.02} 

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_arm2
    pid: {p: 2.0, i: 0.1, d: 0.02}